.github/workflows/rust.yml

```diff
@@ -61,7 +61,7 @@ jobs:
         with:
           rust-version: stable
       - name: Run tests (excluding doctests)
-        run: cargo test --lib --tests --bins
+        run: RUST_MIN_STACK=8388608 cargo test --lib --tests --bins
```

Comment:
To avoid the following test error, add the min stack size of rust:
```
thread 'mdl::test::test_plan_calculation_without_unnamed_subquery' has overflowed its stack.
```